File: Loja de Games/Model/Funcionario.php
<?php
class Funcionario 
{
    public $id;
    public $nome;
    public $endereco;
    public $vendas;
    public $senha;

    public $con;

    public function __construct()
    {
        $conexao = new Conexao();
        $this->con = $conexao->retornarConexao();
    }

    public function Criar()
    {
        $sql = $this->con->prepare("INSERT INTO funcionario (id, nome, endereco, vendas, senha) VALUES (null,?,?,0,?)");
        $sql->execute([$this->nome,$this->endereco,$this->senha]);
    }

    public function ListarUmFuncionario()
    {
        $sql = $this->con->prepare("SELECT * FROM funcionario WHERE nome = ? AND senha = ?");
        $sql->execute([$this->nome,$this->senha]);
        $row = $sql->fetchAll();

        if(sizeof($row) > 0)
        {
            return $row[0];
        }
        else
        {
            return $row;
        }
    }

    public function Login()
    {
        $sql = $this->con->prepare("SELECT * FROM funcionario WHERE nome = ? AND senha = ?");
        $sql->execute([$this->nome,$this->senha]);
        $count = $sql->fetchColumn();

        if($count > 0)
        {
            return true;
        }
        else
        {
            return false;
        }
    }
    
	public function Editar()
    {
        $sql = $this->con->prepare("UPDATE funcionario SET nome = ?, endereco = ?, vendas = ?, senha = ?  WHERE id=?");
		$sql->execute([$this->nome,$this->endereco,$this->vendas,$this->senha,$this->id]);
	}

	public function Deletar()
    {
		$sql = $this->con->prepare("DELETE FROM funcionario WHERE id=?");
		$sql->execute([$this->id]);
	}
}
?>

File: C#/loja_de_games/loja_de_games/Classes/Funcionario.cs
namespace loja_de_games.Classes
{
    public static class Funcionario
    {
        public static int id { get; set; }
        public static string nome { get; set; }
        public static int vendas { get; set; }
        public static string endereco { get; set; }
        public static string senha { get; set; }
        public static Conexao con = new Conexao();

        public static void Criar()
        {
            con.ExecutarComando($"INSERT INTO `funcionario`(`id`, `nome`, `vendas`, `endereco`, `senha`) VALUES (null,{nome},0,{endereco},{senha}) ");
        }

        public static void Editar()
        {
            con.ExecutarComando($"UPDATE `funcionario` SET `nome`= '{nome}',`vendas`= '{vendas}',`endereco`= '{endereco}',`senha`= '{senha}' WHERE id = '{id}'");
        }

        public static void Deletar()
        {
            con.ExecutarComando($"DELETE FROM `funcionario` WHERE id = '{id}'");
        }

        public static bool Login()
        {
            var resultado = con.ExecutarConsulta($"SELECT * FROM `funcionario` WHERE `nome`='{nome}' AND `senha`='{senha}'");

            if (resultado.Rows.Count > 0)
            {
                id = (int)resultado.Rows[0]["id"];
                nome = (string)resultado.Rows[0]["nome"];
                vendas = (int)resultado.Rows[0]["vendas"];
                endereco = (string)resultado.Rows[0]["endereco"];
                senha = (string)resultado.Rows[0]["senha"];

                return true;
            }
            else
            {
                Logoff();
                return false;
            }
        }

        public static void Logoff()
        {
            id = 0;
            nome = "";
            vendas = 0;
            endereco = "";
            senha = "";
        }
    }
}
